src/loaders/sequelize.ts

```diff
@@ -0,0 +1,16 @@
+import { Sequelize } from 'sequelize';
+
+const connectURL = process.env.DB_CONNECT_URL;
+
+export const sequelize = new Sequelize(connectURL, { dialect: 'postgres' });
```

Comment:
this module should export function which receives the destructed parameter `config`. This module config should centralize cooperation with environment settings .

Also, in this module we could place the logic to associate the models with sequalize instance. For example:

```
...
// link all the models
    fs
        .readdirSync(modelsPath)
        .filter(file => {
            return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js');
        })
        .forEach(file => {
            const func = require(`${path.join(modelsPath, file)}`);
            const model = (typeof func === 'function'
                ? func
                : func.default)(sequelize, Sequelize.DataTypes)

 

            models[model.name] = model;
        });

 

    // associate the models
    Object
        .keys(models)
        .forEach(modelName => {
            if (models[modelName].associate) {
                models[modelName].associate(models);
            }
        });
...
```